examples/svelte-example/package.json

```diff
@@ -23,13 +23,13 @@
     "vite": "^5.4.17"
   },
   "dependencies": {
-    "@uppy/core": "workspace:*",
-    "@uppy/dashboard": "workspace:*",
-    "@uppy/drag-drop": "workspace:*",
-    "@uppy/progress-bar": "workspace:*",
-    "@uppy/svelte": "workspace:*",
-    "@uppy/webcam": "workspace:*",
-    "@uppy/xhr-upload": "workspace:*",
+    "@uppy/core": "^4.0.0",
```

Comment:
Unrelated change, let's revert this

Reply:
This was probably me trying to modify the examples to get the StackBlitz embed working on the uppy.io repo, but it ended up in this PR unexpectedly lol , I've reverted it now.